app/javascript/components/pages/ApartmentShow.js

```diff
@@ -27,44 +24,50 @@ const ApartmentShow = ({apartments}) => {
         />
         <CardBody>
           <CardTitle>
-            {selectedApartment.address}
+
+            Address: {selectedApartment.address}
+
           </CardTitle>
           <CardSubtitle>
             Planet: {selectedApartment.planet}
           </CardSubtitle>
           <CardText>
-            Bedrooms: {selectedPlant.bedrooms}
+
+            Bedrooms: {selectedApartment.bedrooms}
           </CardText>
           <CardText>
-            Bathrooms: {selectedPlant.bathrooms}
+            Bathrooms: {selectedApartment.bathrooms}
           </CardText><CardText>
-            square_footage: {selectedPlant.square_footage}
+            square_footage: {selectedApartment.square_footage}
           </CardText><CardText>
-            Price: {selectedPlant.price}
+            Price: {selectedApartment.price}
           </CardText><CardText>
-            Utilities: {selectedPlant.utilities}
+            Utilities: {selectedApartment.utilities}
           </CardText><CardText>
-            Pets: {selectedPlant.pets}
+            Pets: {selectedApartment.pets}
           </CardText><CardText>
-            Parking: {selectedPlant.parking}
+            Parking: {selectedApartment.parking}
+
           </CardText>
         </CardBody>
       </Card>
-      
       )}
       <Button>
       <NavLink 
-      to={`/apartmentedit/${selectedApartment.id}`}>
+
+      to={`/apartmentedit/${selectedApartment?.id}`}>
+
           Edit Apartment
       </NavLink>
       </Button>
       <Button>
       <NavLink 
       to={`/apartmentindex`}>
-          Back to Apartment Index
+
```

Comment:
Still have a few blank lines, can you take care of those before you merge, thanks!